action.yml

```diff
@@ -8,8 +8,12 @@ inputs:
     description: "Directory that contains the files you want to minify. Defaults to current directory (.)"
     required: false
     default: "."
+  exclude_directories:
+    description: "Exclude child folders from the main directory. Combine them in a string separated by semicolon and relative to the main directory eg. 'dir1;js/dir2;js/test/dir3'"
+    required: false
+    default: ""
```

Comment:
Hey @alexmigf! 

Can you switch this please to use line breaks?

Here's an example from StackOverflow.
https://stackoverflow.com/a/75420778/5512292